dwm.c

```diff
@@ -479,6 +492,9 @@ struct Monitor {
 	#if BAR_ALTERNATIVE_TAGS_PATCH
 	unsigned int alttag;
 	#endif // BAR_ALTERNATIVE_TAGS_PATCH
+	#if BAR_COLORFULTAGS_PATCH
+	unsigned int colorseltag;
```

Comment:
exactly, I just wrote it for the sake of the patch. I guess it would be more logical to have it as a global constant.

Apart from that, we could completely remove this variable and just turn it on in the code whenever underline tags are enabled. Since this is dwm-flexipatch we're talking about, I guess it would be reasonable to consider this case.

Reply:
@bakkeby please let me know if we should keep this variable. I personally don't see any reason for it being there. I just included it for the sake of the patch.